ui/send_page.go

```diff
@@ -754,9 +778,29 @@ func (pg *SendPage) watchForBroadcastResult() {
 	}
 }
 
+// changeEvt tracks change event on the editor and adjust its width of the ssend amount input field
+// pg.width is the widith of a single text character
 func (pg *SendPage) changeEvt(evt widget.EditorEvent) {
+	textLength := pg.sendAmountEditor.Editor.Len()
+	n := 29
+
+	// cal decrements the pg.width for a particular range of values
+	cal := func(val int) int {
+		return n - val
+	}
 	switch evt.(type) {
 	case widget.ChangeEvent:
+		textLength = textLength + 1
+		switch {
+		case pg.width <= 80:
+			pg.width = textLength * cal(0)
+		case pg.width > 80 && pg.width < 137:
```

Comment:
*  why are those specific values, 80, 137 and 167 used? the comment doesn't state that. 
*  "handle" would be a better description for what changeEvt does, rather than track it.
*  also, "ssend" in the comment.
*  you can also state very briefly why the editor width is adjusted on a change event